src/core/maps/transferable_belief_model.h

```diff
@@ -0,0 +1,156 @@
+#ifndef SLAM_CTOR_CORE_TBM_H
+#define SLAM_CTOR_CORE_TBM_H
+
+#include <cassert>
+#include <vector>
+#include <string>
+
+/*------------------ declaration ---------------------*/
+
+//transferable belief model
+class TBM {
+public:
+  //public static belief types
+  static enum class Belief {
+    UNKNOWN,    //00
```

Comment:
Please, assign the values explicitly (e.g. 0b00) since correctness of some operations depends on these values.